### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the "Report Review" menu items

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#58]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Report Review" symbol available under the "User Reviews" section (The flag Symbol)
3. Press Enter key in the keyboard.
4. Observe the "Name" property of the list box in Inspect

**Actual Result:**
The "Name property is empty
Narrator Behaviour:
Narrator announces as "Menu"

Expected Result:
The "Name" property should be specific
Eg:
"Report Review" Menu

User Impact:
Visually impaired users  will not get to know the purpose of the menu items as the screen reader will not announce anything when the user arrives at the menu items
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

Screenshot:
![name property is empty for the report review menu items](https://cloud.githubusercontent.com/assets/24825441/21758967/81cf48d2-d666-11e6-83cf-c82cf27867c2.png)

**Comment by waderyan** [comment]
Marketplace issue.
